colossalai/nn/layer/moe/_operation.py

```diff
@@ -4,11 +4,11 @@
 from typing import Any, Tuple, Optional
 from torch.distributed import ProcessGroup
 
-U_CUDA_MODE = False
+KERNEL_OPTIM = False

```

Comment:
USE_KERNEL_OPTIM

Reply:
Whether uses kernel is controled by use_kernel in MoeLayer. KERNEL_OPTIM only shows whether CUDA kernel optimization can be used.

Reply:
you name is misleading. I suppose KERNEL_OPTIM is an optimizer instance. Change the name and make it explains itself.

Reply:
I use COL_MOE_KERNEL instead.

Reply:
COL_MOE_KERNEL_FLAG? it's a bool var. you name should reflect its type.

Reply:
done